src/components/access-type.js

```diff
@@ -486,74 +418,87 @@ class AccessTypeBase extends React.Component {
       accessIsLoading: this.props.accessIsLoading,
       getAssetPlans: this.props.getAssetPlans,
       validateCoupon: this.validateCoupon,
-      cancelSubscription: this.cancelSubscription,
+      cancelSubscription: this.cancelSubscription
     });
   }
 }
 
 AccessTypeBase.propTypes = {
   children: PropTypes.func.isRequired,
+
+  /** Email address of the user */
```

Comment:
we defined the prooTypes. It will take care.  No need to explicitly mention.
![image](https://user-images.githubusercontent.com/37729206/126615333-2c2d8d1f-1d6f-4c2b-9c57-a24e469dcc74.png)